fdbclient/BackupContainerFileSystem.actor.cpp

```diff
@@ -609,10 +642,11 @@ class BackupContainerFileSystemImpl {
 			s.restorable = true;
 			// If this is not a single-version snapshot then see if the available contiguous logs cover its range
 			if (s.beginVersion != s.endVersion) {
-				if (!desc.minLogBegin.present() || desc.minLogBegin.get() > s.beginVersion)
-					s.restorable = false;
+				s.restorable = hasContinuousLogsForSnapshot(logs, s.beginVersion, s.beginVersion);
```

Comment:
IMHO, this only works for For old backup mutation logs. For partitioned logs, we probably can use `isPartitionedLogsContinuous()`.